Makefile

```diff
@@ -189,6 +219,15 @@ release-major:
 	cargo run -p kamu-repo-tools --bin release -- --major
 
 
+###############################################################################
+# Generated resources
+###############################################################################
+
+.PHONY: resources
+resources:
```

Comment:
Not really grasping the difference with the next section (`# Codegen`):

Q: What do you think to rename this target to `codegen-resources`?

Reply:
`codegen` to me is a step that updates inputs to the compilation

`resources` I seperated as a step that produces outputs from the code

I don't see ever wanting to do both steps at once, so I decided to keep them separate.

Not super happy with the name, but rolled with it just because of the current directory name where these outputs live.